crmsh/ui_configure.py

```diff
@@ -962,11 +962,14 @@ def do_commit(self, context, arg0=None, arg1=None):
     @command.skill_level('administrator')
     @command.completers(compl.choice(['force']))
     def do_upgrade(self, context, force=None):
-        "usage: upgrade [force]"
-        if force and force != "force":
-            context.fatal_error("Expected 'force' or no argument")
+        "usage: upgrade <force>"
+        if not force or force != "force":
+            context.fatal_error("'force' option is required")
         cib_factory.ensure_cib_updated()
-        return cib_factory.upgrade_validate_with(force=config.core.force or force)
```

Comment:
With this change, `crm -f upgrade` will not work any longer. This changes the behavior of a released branch.

Reply:
Changed to support -F option